agent/api/task/task_linux.go

```diff
@@ -53,16 +53,16 @@ const (
 	internalExecCommandAgentCertVolumeName      = internalExecCommandAgentNamePrefix + "-tls-cert"
 
 	// TODO: [ecs-exec] decide if this needs to be configurable or put in a specific place in our optimized AMIs
-	execCommandAgentHostBinDir           = "/home/ec2-user/ssm-agent/linux_amd64"
-	execCommandAgentBinName              = "amazon-ssm-agent"
-	execCommandAgentSessionWorkerBinName = "ssm-session-worker"
-	execCommandAgentSessionLoggerBinName = "ssm-session-logger"
+	ExecCommandAgentHostBinDir           = "/home/ec2-user/ssm-agent/linux_amd64"
+	ExecCommandAgentBinName              = "amazon-ssm-agent"
+	ExecCommandAgentSessionWorkerBinName = "ssm-session-worker"
+	ExecCommandAgentSessionLoggerBinName = "ssm-session-logger"
```

Comment:
The session-logger is not necessary for linux.

Reply:
removed